students/Gentner/Phys333_Finalproj/Phys333_Finalproj.ino

```diff
@@ -0,0 +1,101 @@
+//state machine
+
+//Case 1 - If in bright light then do nothing
+  //Do nothing
+//Case 2 - If photoresitor detects low light then turn on IR led
+  //Turn on IRLED
+  //Case 1 - If Detector reads IR then do nothing
+    //Do Nothing
+  //Case 2 - Break in IR 
+    //Check Debounce setting
+    //If Debounce is true 
+    //turn on Lights and do not check photoresistor
+    //Leave on for ------
+    //Turn off LED 
+    //Return to top
+
+const int Lights=12;
+const int PR=11;
+const int IR=4;
+const int IRD=A0;
+boolean lastDetect = LOW; // variable containing the previous Detect state
+boolean currentDetect = LOW; // variable constaing the current Detect state
+unsigned int ledMode = 1; // present state of the Blue Light
+
+
+void setup() {
+pinMode(IR,OUTPUT);
+pinMode(IRD,INPUT);
+Serial.begin(9600);
+}
+
+void loop() {
+int DetectRead = 0;
+int Detect = 0;
+
+
+  //Detector Debounce
+// {
+  //   currentDetect = debounce(lastDetect);             //Read debounced state
+  //   if (lastDetect >= 1000 && currentDetect <= 1015)  //if it was pressed...
+  //   {
+  //     ledMode++;  //Increase the LED state value
+  //     Serial.print("LedMode-");
+  //     Serial.print(ledMode);
+  //     Serial.print(" ");
+  //   }
```

Comment:
The beauty of git is that you don't need to keep commented stuff in like this. Commit it, then delete and commit again. You can always retrieve it from history.